lib/screens/profile/faq.dart

```diff
@@ -1,110 +1,76 @@
+import 'package:brunstadtv_app/graphql/client.dart';
 import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
-import 'package:auto_route/auto_route.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
 
+import '../../components/custom_back_button.dart';
+import '../../components/custom_tab_bar/custom_tab_bar.dart';
+import '../../components/status_indicators/error_generic.dart';
+import '../../components/status_indicators/loading_generic.dart';
 import '../../components/web/dialog_on_web.dart';
 import '../../theme/design_system/design_system.dart';
-
+import '../../graphql/queries/faq.graphql.dart';
 import '../../l10n/app_localizations.dart';
 
-class FAQItem {
-  final int id;
-  final String question;
-  final String answer;
-
-  FAQItem({
-    required this.id,
-    required this.question,
-    required this.answer,
-  });
-}
-
-class FAQScreen extends StatefulWidget {
+class FAQScreen extends HookConsumerWidget {
   const FAQScreen({super.key});
 
   @override
-  State<FAQScreen> createState() => _FAQScreenState();
-}
+  Widget build(BuildContext context, WidgetRef ref) {
+    final refreshKey = useState(UniqueKey());
+    final categoriesFuture = useMemoized(
```

Comment:
Can you change to useState(useMemoized())?
See: https://github.com/bcc-code/bcc-media-app/commit/a13d587e7d7443c833aa4ac9f7574ff388d91702
Its a bit simpler and easier to read